index.html

```diff
@@ -21,17 +21,40 @@ <h1 class="mh-logo">
                 
             </h1>
     
-              <h1 class="button">
-                  <a href="#"><img alt="hambutton" src="./images/hambutton.svg"></a>
-              </h1>
+             
+              <div class="desktop-buttons">
+                <ul class="main-nav-list">
+                  <li>
+                      <a href="#">Portfolio</a>
+                  </li>
+                  <li>
+                      <a href="#">About</a>
+                  </li>
+                  <li>
+                      <a href="#">Contact</a>
+                  </li>
+                  <li>
+                      <a href="#"><img src="images/mail.svg" alt=""></a>
+                  </li>
+              </ul>
+              </div>
+              <div class="ham-button">
+                <a href="#"><img alt="hambutton" src="./images/hambutton.svg"></a>
+            </div>
         </div>
 
     </header>
 
 
     <section class="body-container">
-   
 
+   
+      <section class="intro-container">
+        <ul class="socials">
+          <li><a href='https://github.com/FabricioLatl/'><img alt="github" src='images/github.svg' /></a></li>
+          <li><a href='https://www.linkedin.com/in/fabriciolatl/'><img alt="linkedin" src='images/linkedin.svg' /></a></li> 
+          <li><a href='https://twitter.com/Wafer_lt'><img alt="twitter" src='images/twitter.svg' /></a></li>
+        </ul>
       <ul class="intro">
         <li>
           <h1 class="name">
```

Comment:
- [ ] Kindly make sure your intro section sticks to the Figma design as required for the project(fonts on text(weight, size,...), background color, text color, spacing(top && bottom), missing icons,...) Make sure you check every missing detail to stick to the Figma template.
### Figma design
![Screenshot from 2022-07-16 16-24-42](https://user-images.githubusercontent.com/83042742/179359197-0770abc4-d228-489d-84d1-817197e1442b.png)

### Your design
![Screenshot from 2022-07-16 16-36-03](https://user-images.githubusercontent.com/83042742/179359357-90975466-fdac-4aa9-befc-d6ba4dac1f81.png)